server/secrets.go

```diff
@@ -0,0 +1,98 @@
+package server
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/docker/docker/pkg/archive"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/selinux/go-selinux/label"
+	"github.com/pkg/errors"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+)
+
+const (
+	// defaultmountCfgFile holds the default mount paths in the form "host:container"
+	defaultMountCfgFile = "/usr/share/containers/mounts.conf"
+	// overridetmountCfgFile holds the override mount paths in the form "host:container"
+	overrideMountCfgFile = "/etc/containers/mounts.conf"
+)
+
+// readMountFile returns a list of the host:container paths
+func readMountFile(mountFilePath string) ([]string, error) {
+	var mountPaths []string
+	file, err := os.Open(mountFilePath)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	scanner.Split(bufio.ScanLines)
+	for scanner.Scan() {
+		mountPaths = append(mountPaths, scanner.Text())
+	}
+
+	return mountPaths, nil
+}
+
+// getHostAndCtrDir separates the host:container paths
+func getHostAndCtrDir(path string) (string, string, error) {
+	arr := strings.SplitN(path, ":", 2)
+	if len(arr) == 2 {
+		return arr[0], arr[1], nil
```

Comment:
need to validate these

Reply:
as in check if the paths exist?

Reply:
yeah, just on the host-side, and emit a warning that we're skipping that if it doesn't exist on host

Reply:
fixed.